en/docs/install-and-setup/upgrading-wso2-api-manager/upgrading-from-1100-to-400.md

```diff
@@ -4608,3 +4608,45 @@ This concludes the upgrading process.
     If you are using a migrated API and need to consume it via an application that supports JWT authentication (default type in API-M 3.2.0), you need to republish the API. JWT authentication will not work if you do not republish the API, as it looks for a local entry that will get populated while publishing.
 
     You can consume the migrated API via an OAuth2 application without an issue.
+
```

Comment:
we have not added these notes in 1.10.0 and 1.9.1 docs as they are anyway redirecting to 2.0.0 -> 4.0.0 migration doc. Please check with @Mariangela whether we need to add in old versions. If yes, we will have to add all the notes we have added in latest migrations docs.

Reply:
@Mariangela . Your thoughts on this?